### ISSUE TITLE
Dualsense - rear audio input (for haptic feedback) cutting off when connecting through DS4Windows.

### ISSUE THREAD

**Issue by AndrzejSzpadel** [Ryochan7/DS4Windows#2252]
**Describe the bug**
Trying to use Dualsense rear audio feature to have haptic feedback based on audio cuts off when connecting controller through usb with DS4Windows running in the background

**To Reproduce**
1. Using software like voicemeeter to combine channels on Dualsense and output audio to eg. tv at the same time
2. Connect controller when DS4 Windows is running
3. Haptic Feedback cuts off when connected

**Expected behavior**
Haptic feedback continue to work with DS4Windows running

**Solution to get haptic feedback working again**
Disconnect Controller, exit Voicemeeter and exit DS4Windows, uninstall wireless controller driver and connect controller again, 
run Voicemeeter software without DS4 Windows.

**Desktop:**
 - Controller Make and Model: Sony Midnight Black Dualsense (CFI-ZCT1W)
 - OS: Windows 10 Pro Build 21H1
 - DS4Windows Version 3.0.10

**Additional context**
Basically trying to get haptic feedback functionality through audio with xinput functionality at the same time and seems like DS4Windows is blocking audio or haptic motors when controller is connected, no matter if its in exclusive mode or not
